qvm/sys-gui-xfce.sls

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+##
+# qvm.sys-gui-xfce
+# ================
+##
+
+sys-gui-xfce:
+  pkg.installed:
+    - pkgs:
+        - qubes-vm-guivm
+        - qubes-manager
+        - xfwm4
+        - garcon
+        - xfce4-settings-qubes
```

Comment:
I think this should take advantage of a XFCE package group, otherwise a lot of packages will be missing (panel widgets, maybe even xfce4-session). Check Fedora groups (like "XFCE" one), but if it's too large, check the one we have in dom0 - it's in https://github.com/QubesOS/qubes-qubes-release/blob/master/comps-qubes.xml